pkg/gen/input/makefile/internal/file/Makefile.gen.app.mk.template

```diff
@@ -6,7 +6,7 @@ YQ=docker run --rm -u $$(id -u) -v $${PWD}:/workdir mikefarah/yq:4.29.2
 HELM_DOCS=docker run --rm -u $$(id -u) -v $${PWD}:/helm-docs jnorwood/helm-docs:v1.11.0
 
 ifdef APPLICATION
-DEPS := $(shell ls $(APPLICATION)/charts)
+DEPS := $(shell find $(APPLICATION)/charts -name "Chart.yaml" -printf "%h\n")
```

Comment:
`-printf` is not supported on macOS – can we please use something else instead that is cross-platform?